cpu/sensors.py

```diff
@@ -33,6 +33,8 @@ class Sensors(Test):
 
     """
     Run sensors command on ppc architectures.
+
+    :avocado: tags=cpu,power,privileged
```

Comment:
@clebergnu @PraveenPenguin  is right. I guess docstring needs a change. This is not a ppc specific test. Thanks.